lightning/src/ln/chan_utils.rs

```diff
@@ -698,6 +698,37 @@ pub fn build_htlc_transaction(commitment_txid: &Txid, feerate_per_kw: u32, conte
 	}
 }
 
+/// Returns the witness required to satisfy and spend a HTLC input.
+pub fn build_htlc_input_witness(
```

Comment:
how come we need this method? Didn't we have the ability to spend an HTLC output before?